test-integration/apps/beacon-consumer/routes/analytics.js

```diff
@@ -7,8 +7,8 @@ module.exports = ({ beaconFilePath, logger }) => {
         const beacon = req.body;
 
         // UNCOMMENT FOR MORE LOGGING FOR TROUBLESHOOTING TESTS
-        // logger(`Received beacon:`);
-        // logger(JSON.stringify(beacon, null, 2));
+        logger(`Received beacon:`);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/groupby/gb-tracker-client/pullRequest?prid=8003520)
```suggestion
        logger("Received beacon:");
```